helpdesk_mgmt/views/helpdesk_ticket_menu.xml

```diff
@@ -17,6 +17,9 @@
             <field name="res_model">helpdesk.ticket</field>
             <field name="view_mode">tree,kanban,form,pivot</field>
             <field name="view_type">form</field>
+            <field name="context">
+              {'search_default_open': 1}
```

Comment:
Put all in one line for avoiding unneeded paddings.

Reply:
I will fix it.